modules/monitoring-excluding-a-user-defined-project-from-monitoring.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+[id="excluding-a-user-defined-project-from-monitoring_{context}"]
+= Excluding a user-defined project from monitoring
+
+Individual user-defined projects can be excluded from user workload monitoring. To do so, simply add the `openshift.io/user-monitoring` label to the project's namespace with a value of `false`.
+
+.Procedure
+
+. Add the label to the project namespace:
++
+[source,terminal]
+----
+$ oc label namespace my-project 'openshift.io/user-monitoring=false'
+----
++
+. To reenable monitoring, remove the label from the namespace:
```

Comment:
s/reenable/re-enable, per the IBM style guidelines

Reply:
Fixed. Thanks!